storage/id_alloc.go

```diff
@@ -56,10 +68,11 @@ func NewIDAllocator(idKey proto.Key, db *client.KV, minID int64, blockSize int64
 		ids:       make(chan int64, blockSize+blockSize/2+1),
 	}
 	ia.ids <- allocationTrigger
-	return ia
+	return ia, nil
 }
 
-// Allocate allocates a new ID from the global KV DB. If multiple
+// Allocate allocates a new ID from the global KV DB, and it will
```

Comment:
I think you can leave out the everything after the comma (it refers to an unexported function).